components/slot-fill/slot.js

```diff
@@ -6,18 +6,32 @@ import { noop, map, isString, isFunction } from 'lodash';
 /**
  * WordPress dependencies
  */
-import { Component, Children, cloneElement } from '@wordpress/element';
+import {
+	Component,
+	Children,
+	Fragment,
+	cloneElement,
+	createRef,
+} from '@wordpress/element';
 
 class Slot extends Component {
 	constructor() {
 		super( ...arguments );
 
-		this.bindNode = this.bindNode.bind( this );
+		this.placeholderRef = createRef();
 	}
 
 	componentDidMount() {
 		const { registerSlot = noop } = this.context;
 
+		if ( this.props.bubblesVirtually ) {
+			this.node = this.placeholderRef.current.parentNode;
```

Comment:
I'm a little bit concerned about this. What happens if we do:

```js
<div>
    <Slot/>
     <somethingElse />
</div>
```

What happens when the `somethingElse` rerenders but not the portal (or the opposite)? I wonder how React handles this and if it's safer to just keep the wrapper for the `bubblesVirtually` case, do we need to remove it for this use-case as well, it seems the main issue is for Plugin APIs which don't use `bubblesVirtually`?